source/topic4.rst

```diff
@@ -109,6 +109,25 @@ Abstraction
 Data Structures
 ===============
 
+* A data structure is an actual implementation of an ADT
+
+* Using the `ContactList` example, the *what* would be:
+    * Keep track of ``Friend`` objects
+    * Add and remove ``Friend`` objects
+    * Get a ``Friend``
+    * ...
+
+* And, the *how* would be the actual code we see
+    * We used an array to keep track of the ``Friend`` objects
+    * Do a linear search through the array to find a specific ``Friend``
+
+* A large focus in this class will be both the *what* and the *how and how to keep them seperate
+    * The interface and the implementation
```

Comment:
Other terms you could throw out there are the _abstract type_ and the _concrete type_.

Reply:
I may save these until I get to interfaces & inheritance and whatnot.